[CI] ReindexDoubleRelocationIT testDoubleRelocationWithManagementEndpoints failing

**Build Scans:**
- [elasticsearch-intake #39181 / part1](https://gradle-enterprise.elastic.co/s/pjoqt2oyil2p6)
- [elasticsearch-intake #39176 / part1](https://gradle-enterprise.elastic.co/s/vudm6qr6xpxu6)
- [elasticsearch-pull-request #143435 / part-1](https://gradle-enterprise.elastic.co/s/r74u26fcodngo)
- [elasticsearch-pull-request #143425 / part-1](https://gradle-enterprise.elastic.co/s/tnfvgk7gr5b5o)
- [elasticsearch-pull-request #143414 / part-1](https://gradle-enterprise.elastic.co/s/zjrt4taby7owa)
- [elasticsearch-pull-request #143394 / part-1](https://gradle-enterprise.elastic.co/s/vgibowhk356vs)
- [elasticsearch-pull-request #143323 / part-1](https://gradle-enterprise.elastic.co/s/e5mjntyqb3xhe)
- [elasticsearch-pull-request #143302 / part-1](https://gradle-enterprise.elastic.co/s/vvnj45u74qczg)
- [elasticsearch-pull-request #143294 / part-1](https://gradle-enterprise.elastic.co/s/j3aquh36bsoqy)

**Reproduction Line:**
```
./gradlew ":modules:reindex-management:internalClusterTest" --tests "org.elasticsearch.reindex.management.ReindexDoubleRelocationIT.testDoubleRelocationWithManagementEndpoints" -Dtests.seed=985181D893FDEAAA -Dtests.locale=lb-LU -Dtests.timezone=Asia/Rangoon -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.reindex.management.ReindexDoubleRelocationIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testDoubleRelocationWithManagementEndpoints'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: "by user request"
     but: was null
```

**Issue Reasons:**
- [main] 1 consecutive failures in test testDoubleRelocationWithManagementEndpoints
- [main] 1 consecutive failures in step part1
- [main] 1 consecutive failures in pipeline elasticsearch-intake
- [main] 9 failures in test testDoubleRelocationWithManagementEndpoints (7.8% fail rate in 116 executions)
- [main] 2 failures in step part1 (7.7% fail rate in 26 executions)
- [main] 7 failures in step part-1 (8.1% fail rate in 86 executions)
- [main] 2 failures in pipeline elasticsearch-intake (7.7% fail rate in 26 executions)
- [main] 7 failures in pipeline elasticsearch-pull-request (8.9% fail rate in 79 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/modules/reindex-management/src/internalClusterTest/java/org/elasticsearch/reindex/management/ReindexDoubleRelocationIT.java
+++ b/modules/reindex-management/src/internalClusterTest/java/org/elasticsearch/reindex/management/ReindexDoubleRelocationIT.java
@@ -198,7 +198,7 @@ public class ReindexDoubleRelocationIT extends ESIntegTestCase {
         assertThat(afterCancelMap.get("original_task_id"), is(nullValue()));
         assertThat(afterCancelMap.get("original_start_time_in_millis"), is(nullValue()));
         assertThat(ObjectPath.eval("status.canceled", afterCancelMap), equalTo("by user request"));
-        assertThat(ObjectPath.eval("response.canceled", afterCancelMap), equalTo("by user request"));
+        // depending on how quickly we cancel, we might either get an error (shard failure) or response, so not asserting
     }
 
     private GetReindexResponse getReindex(final TaskId taskId, final boolean waitForCompletion) {

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -509,9 +509,6 @@ tests:
 - class: org.elasticsearch.xpack.security.QueryableReservedRolesIT
   method: testConfiguredReservedRolesAfterClosingAndOpeningIndex
   issue: https://github.com/elastic/elasticsearch/issues/147972
-- class: org.elasticsearch.reindex.management.ReindexDoubleRelocationIT
-  method: testDoubleRelocationWithManagementEndpoints
-  issue: https://github.com/elastic/elasticsearch/issues/148462
 - class: org.elasticsearch.xpack.stateless.reshard.StatelessReshardIT
   method: testReshardMetrics
   issue: https://github.com/elastic/elasticsearch/issues/148463

